packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx

```diff
@@ -147,7 +151,8 @@ export const ChatMessageActionFlyout = (props: ChatMessageActionFlyoutProps): JS
         },
         submenuIconProps: {
           iconName: 'HorizontalGalleryRightButton',
-          styles: menuIconStyleSet
+          styles: menuIconStyleSet,
+          id: 'chat-composite-message-contextual-menu-read-info'
```

Comment:
if ids are added for UI tests, use data-ui-id. This likely won't work for adding to the icon of the menu item - can you add to the menu item itself?

Reply:
Something like this should work:
![image](https://user-images.githubusercontent.com/2684369/162265602-be6a4759-4f55-48e7-94d9-6838e6d29263.png)

Reply:
We chatted offline and found there isn't a good solution for this, lets keep id